website/partials/_download_box.ejs

```diff
@@ -20,20 +20,20 @@
                     </thead>
                     <tbody>
                     <tr>
-                        <td><a href="https://www.apache.org/dyn/closer.lua?action=download&filename=incubator/streampipes/0.70.0/apache-streampipes-0.70.0-incubating-source-release.zip">apache-streampipes-0.70.0-incubating-source-release.zip</a></td>
+                        <td><a href="https://www.apache.org/dyn/closer.lua?action=download&filename=streampipes/0.70.0/apache-streampipes-0.70.0-incubating-source-release.zip">apache-streampipes-0.70.0-incubating-source-release.zip</a></td>
                         <td>0.70.0</td>
                         <td>2022-10-13</td>
                         <td>
-                            <a href="https://downloads.apache.org/incubator/streampipes/0.70.0/apache-streampipes-0.70.0-incubating-source-release.zip.sha512">SHA</a>
```

Comment:
We have not set up the new distribution directories yet.
So in case we want to update the website before the announcement, we either need to copy the last incubating release to the new distribution area, keep the old link or wait for the first non-incubating release and update the website afterwards, whatever you prefer ;-)

Reply:
Please give it a try, at least it has worked already on my end :)

Reply:
Oh sorry, I didn't know that there is a redirect, very cool ;-)